src/sentry/workflow_engine/models/detector_state.py

```diff
@@ -10,6 +10,7 @@
 
 class DetectorStatus(StrEnum):
     OK = "ok"
+    FAILED = "failed"
```

Comment:
I was just taking a look at this whim in process workflows, there it seems like it'd be helpful to have a full mapping to the priority level for an issue group.

How were you thinking those levels would play into this?

Reply:
For metric alerts I think it's ok/warning/error since it's tracking the current state. Unless we just want to use high/medium priority instead.

Reply:
i was thinking we could just map it all the priority level and only have 1 type for everything - if we start to tie it to issue occurrences more that might be helpful.